en/flight_stack/controller_diagrams.md

```diff
@@ -16,6 +16,62 @@ Request access from dev team. -->
 * Depending on the mode, the outer (position) loop is bypassed (shown as a multiplexer after the outer loop). The position loop is only used when holding position or when the requested velocity in an axis is null.
 * The integrator in the inner loop (velocity) controller includes an anti-reset windup (ARW) using a clamping method.
 
+## Fixed-Wing Position Controller
```

Comment:
This looks excellent - I've only scanned the maths, but the explanation of TECS and the control problem is very clear.

TECS is part of the Fixed Wing Position Control module. IFF it is the only part that you intend to document here then the current structure is "OK". If you ever expect to cover the other bits then I would restructure slightly to have the diagram and a description of Fixed Wing Position Control under the parent heading. This makes it easier to provide context for TECs and any other components. So something like:

```
## Fixed-Wing Position Controller

 The fixed wing position control module is used to set vehicle attitude based on desired X, Y, Z. It uses the TECs library to XXX, and the L1 controller to ZZZ. The structure is shown below.

	![TECS](../../assets/diagrams/tecs_in_context.svg)

### Total Energy Control System (TECS)

...
```
Hope that makes sense.